docs/project-model/project-json-reference.md

```diff
@@ -55,6 +39,55 @@
     * [emitEntryPoint](#emitEntryPoint)
     * [xmlDoc](#xmlDoc)
     * [preserveCompilationContext](#preserveCompilationContext)
+    * [compilerName](#compilerName)
+    * [compile](#compile)
+        * [include](#compile-include)
+        * [exclude](#compile-exclude)
+        * [includeFiles](#compile-include-files)
+        * [excludeFiles](#compile-exclude-files)
+        * [builtIns](#compile-builtins)
+        * [mappings](#compile-mappings)
+    * [embed](#embed)
+        * [include](#embed-include)
+        * [exclude](#embed-exclude)
+        * [includeFiles](#embed-include-files)
+        * [excludeFiles](#embed-exclude-files)
+        * [builtIns](#embed-builtins)
+        * [mappings](#embed-mappings)
+    * [copyToOutput](#copytooutput)
+        * [include](#copytooutput-include)
+        * [exclude](#copytooutput-exclude)
+        * [includeFiles](#copytooutput-include-files)
+        * [excludeFiles](#copytooutput-exclude-files)
+        * [builtIns](#copytooutput-builtins)
+        * [mappings](#copytooutput-mappings)
+* [publishOptions](#publishoptions)
+    * [include](#publish-include)
+    * [exclude](#publish-exclude)
+    * [includeFiles](#publish-include-files)
+    * [excludeFiles](#publish-exclude-files)
+    * [builtIns](#publish-builtins)
+    * [mappings](#publish-mappings)
+* [runtimeOptions](#runtimeoptions)
+    * [gcServier](#gcserver)
```

Comment:
Also the structure of `runtimeOptions` has changed. https://github.com/aspnet/Home/issues/1418#issuecomment-221745530